style.css

```diff
@@ -1,4 +1,3 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;700&family=Poppins&display=swap');
```

Comment:
This google fonts import cannot be removed! It's used in the stylesheet

Reply:
Can we have a review now?